### PR TITLE
Fix integration tests

### PR DESCRIPTION
**Pull request by jjanvier**
| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added legacy Behats               | -
| Added acceptance tests            | -
| Added integration tests           | -
| Changelog updated                 | -
| Review and 2 GTM                  | 
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

### CODE REVIEW

File: .ci/Dockerfile [hunk 1]
```diff
@@ -72,7 +72,7 @@ COPY . .
 COPY .ci/php.ini /usr/local/etc/php/
 COPY .ci/vhost.conf /etc/apache2/sites-available/000-default.conf
 
-RUN composer $COMPOSER_COMMAND --no-ansi --optimize-autoloader --no-interaction --no-progress --prefer-dist --no-suggest \
```
Comment (skeleton) [inline .ci/Dockerfile]: why do you remove --optimize-autoloader ?

Comment (skeleton) [reply .ci/Dockerfile]: ok, I just saw the comment of Damien

Comment (jjanvier) [reply .ci/Dockerfile]: read Slack @skeleton ;)

**Comment by jjanvier** [comment]
From @damien-carcel 

> First, you may have heard we had some issues with the Integration tests on master. When running them locally, Benoit discovered that almost half of them had a wrong namespace after having been moved, resulting in fatal errors.
We were quite afraid, as the CI was green nonetheless… Hopefully, the CI was right and not skipping any tests. The explanation is that we perform `composer install` on the CI with the option `--optimize-autoloader`. As a result, PSR-4/PSR-0 rules are converted into classmap rules. As our PSR-0/PSR-4 definitions for the tests is shitty, the tests were still miraculously working. However, if you install or dump the PSR rules without `--optimize-autoloader`, they do not.
@julien.janvier is currently fixing the tests namespaces, and we will remove the `--optimize-autoloader` from the CI for now. It will be back (as it is a good practice for production, so it is for the CI) once our namespaces are clean, once the split of the tests by bounded context is done.

**Pull request merged**